runbotics-orchestrator/src/main/resources/config/liquibase/changelog/20230831151264_changelog.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="depcikm" id="20230831151264-1">
+        <preConditions onFail="MARK_RAN">
+            <and>
+                <tableExists tableName="global_variable"/>
+                <not>
+                    <columnExists tableName="global_variable" columnName="creator_id"/>
+                </not>
+                <not>
+                    <foreignKeyConstraintExists foreignKeyName="fk_global_variable_creator_id" foreignKeyTableName="global_variable" />
+                </not>
+            </and>
+        </preConditions>
+        <addColumn tableName="global_variable">
+            <column name="creator_id" type="BIGINT"  />
+        </addColumn>
+        <addForeignKeyConstraint baseTableName="global_variable"
+                                 baseColumnNames="creator_id"
+                                 constraintName="fk_global_variable_creator_id"
+                                 referencedTableName="jhi_user"
+                                 referencedColumnNames="id"
+                                 deferrable="false"
+                                 initiallyDeferred="false"
+                                 validate="true"
+                                 onDelete="SET NULL"
+        />
```

Comment:
move foreignkey creation to another changeset in case we want to get rid of this in the future